html/captive-portal/lib/captiveportal/PacketFence/DynamicRouting/Module/Authentication/Sponsor.pm

```diff
@@ -110,7 +110,7 @@ sub do_sponsor_registration {
     foreach my $key (qw(firstname lastname telephone company sponsor)) {
         $info{$key} = $self->request_fields->{$key};
     }
-    $info{'subject'} = $self->app->i18n_format( "%s: Guest access request", $Config{'general'}{'domain'} );
+    $info{'subject'} = [ "%s: Guest access request", $Config{'general'}{'domain'} ];
```

Comment:
This would need to be internationalized in pf::activation::create_and_send_activation_code since it is used for send email